generators/gae/index.js

```diff
@@ -363,17 +366,41 @@ module.exports = class extends BaseGenerator {
                 });
             },
 
+            askIfCloudSqlIsNeeded() {
+                if (this.abort) return;
+                const done = this.async();
+                const prompts = [];
+
+                prompts.push({
+                    type: 'input',
+                    name: 'gaeCloudSQLInstanceNeeded',
+                    message: 'Initialize a new Cloud SQL instance (Y/N) ?',
+                    default: this.gaeCloudSQLInstanceNeeded ? this.gaeCloudSQLInstanceNeeded : 'Y',
+                    validate: input => {
+                        if (input !== 'Y' && input !== 'N') {
+                            return 'Input should be Y or N';
+                        }
+                        return true;
+                    }
+                });
+
+                this.prompt(prompts).then(props => {
+                    this.gaeCloudSQLInstanceNeeded = props.gaeCloudSQLInstanceNeeded;
+                    done();
+                });
+            },
+
             askForCloudSqlInstance() {
+                if (this.gaeCloudSQLInstanceNeeded === 'N') return;
                 if (this.abort) return;
                 if (this.prodDatabaseType !== 'mysql' && this.prodDatabaseType !== 'mariadb' && this.prodDatabaseType !== 'postgresql')
                     return;
 
                 const done = this.async();
 
                 const cloudSqlInstances = [{ value: '', name: 'New Cloud SQL Instance' }];
-
                 exec(
-                    `gcloud sql instances list  --format='value[separator=":"](project,region,name)' --project="${this.gcpProjectId}"`,
+                    `gcloud sql instances list  --format="value[separator=":"](project,region,name)" --project="${this.gcpProjectId}"`,
```

Comment:
I'm pretty sure there is a mistake here with the `"`

Reply:
Thanks. Corrected 😄